src/shared/services/fields/fields.utils.ts

```diff
@@ -159,6 +163,10 @@ export class FieldsUtils {
     return fieldValue ? FieldsUtils.currencyPipe.transform(fieldValue / 100, 'GBP', 'symbol') : fieldValue;
   }
 
+  private static getLabel(fieldValue: CaseField) {
```

Comment:
can we have the return type, i.e string eclared.

Reply:
good spot